### DISCUSSION TITLE
Badge vs Tag

### DISCUSSION THREAD

**Discussion by hazem3500** [chakra-ui/chakra-ui#2109] [General]
Is there any difference between using a `Badge` and `Tag`, Can't `Tag` cover all the use-cases that `Badge` uses?

**Comment by laurensnl** [comment]
Other than a difference in appearance, they seem pretty much the same. 

<img width="98" alt="Screenshot 2020-10-11 at 11 51 23" src="https://user-images.githubusercontent.com/4354382/95675507-1cccbd80-0bb8-11eb-808e-c8491ebcea8f.png">

It would be nice if the badge adds a little more functionality like Ant Design's badge does:
https://ant.design/components/badge/

**Comment by chasinhues** [comment]
Judging by the explanation in the docs for the two components, the intended use cases for these components are that:
- `Badge`'s are for highlighting an items status for quick recognition
- `Tag`'s are for organising/labeling/categorising
